client.go

```diff
@@ -108,14 +101,14 @@ func (c *Client) newRequest(method string, action string, params Params, body in
 	return req, err
 }
 
-// do takes a prepared API request and makes the API call to Recurly.
+// Do takes a prepared API request and makes the API call to Recurly.
 // It will decode the XML into a destination struct you provide as well
 // as parse any validation errors that may have occurred.
 // It returns a Response object that provides a wrapper around http.Response
 // with some convenience methods.
-func (c *Client) do(req *http.Request, v interface{}) (*Response, error) {
+func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
```

Comment:
Unexported in [5238e2](https://github.com/blacklightcms/go-recurly/pull/10/commits/322b61900754b8bcf3719fcf55a8b1a7955238e2)